Sentinel/Classes/EmailSender/EmailSenderTool.swift

```diff
@@ -79,17 +79,30 @@ public extension EmailSenderTool {
         if EmailSenderView.canSendEmail() {
             EmailSenderView(mailData: getter())
         } else {
-            VStack(spacing: 10) {
-                Text(alertTitle)
-                    .font(.system(size: 17, weight: .bold))
-                    .multilineTextAlignment(.center)
-
-                Text(alertMessage)
-                    .font(.system(size: 14, weight: .regular))
-                    .multilineTextAlignment(.center)
-            }
-            .frame(maxHeight: .infinity, alignment: .top)
-            .padding()
+            ErrorView(alertTitle: alertTitle, alertMessage: alertMessage)
         }
     }
+
+}
+
+// MARK: - ErrorView
+
+private struct ErrorView: View {
```

Comment:
I would recommend putting this into a new file and make it `internal` (no need to add that modifier explicitly), but to make it visible to internal implementation (e..g, if needed in other modules).